operate/client/src/modules/bpmn-js/BpmnJS.ts

```diff
@@ -203,6 +206,21 @@ class BpmnJS {
       this.#hasOuterBorderOnSelection = hasOuterBorderOnSelection;
     }
 
+    // handle op-highlighted flow node markers
+    if (!isEqual(this.#highlightedFlowNodeIds, highlightedFlowNodeIds)) {
+      // remove previous markers
+      this.#highlightedFlowNodeIds.forEach((flowNodeId) => {
+        this.#removeMarker(flowNodeId, 'op-highlighted');
+      });
+
+      // add new markers
+      highlightedFlowNodeIds.forEach((element) => {
+        this.#addMarker(element, 'op-highlighted');
```

Comment:
:bulb: This uses the `addMarker` method from the bpmn-js library. It can be used to add classes to add classes to svg elements in the svg tree. This is then used to add styled to the svg element.

![image](https://github.com/camunda/camunda/assets/2543743/63e16e26-1a9a-48a6-8239-3118fd6f1e33)